packaging/installer/netdata-updater.sh

```diff
@@ -210,6 +210,10 @@ tmpdir=
 
 trap cleanup EXIT
 
+# Random sleep to aileviate stampede effect of Agents upgrading
+# and disconnecting/reconnecting at the same time (or near to).
+sleep $(((RANDOM % 10) + 1))s
+
```

Comment:
On further thought, I'm very strongly of the opinion that this should:

1. Be used only when run non-interactively (no controlling terminal for the script is sufficient IMO).
2. Be _much_ longer on the maximum, probably at least 5 minutes, but more ideally closer to half an hour or an hour.

Reply:
probably we need something like that

```
min = x
max = y
sleep = rand(max - min) + min
```

Reply:
Provided it's a uniform distribution (which I believe is the case for the bash implementation of `$RANDOM`), we should just set the minimum at 0.

Reply:
Unclear of the unformity of Bash's `$RANDOM` but the man page has this to say:

>        RANDOM Each time this parameter is referenced, a random integer between 0 and 32767 is generated.  The sequence of random numbers may  be
>              initialized  by  assigning  a  value  to  RANDOM.  If RANDOM is unset, it loses its special properties, even if it is subsequently
>              reset.

Reply:
Wait, so unsetting `$RANDOM` is a security concern? Like a shell could export `$RANDOM` to a subshell and kill the randomness?

Reply:
@mfundul Yes. Most 'special' shell variables are only special until they're overridden by the environment. It's kind of like how some old languages let you use purely numeric identifiers.

Reply:
Done